lib/pages/settings_pages/settings_page.dart

```diff
@@ -134,18 +140,57 @@ class _SettingsPageState extends State<SettingsPage> {
             SizedBox(
               height: 10,
             ),
-            SettingsMenuWidget(
-                hashtag == null
-                    ? MyLocalizations.of(context, 'enable_auto_hashtag')
-                    : MyLocalizations.of(context, 'disable_auto_hashtag'), () {
-              _manageHashtag();
-            }),
-            SizedBox(
-              height: 35,
-            ),
-            SizedBox(
-              height: 60,
-            ),
+            Container(
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(5),
+                color: Colors.white,
+                border: Border.all(color: Colors.black.withOpacity(0.1)),
+              ),
+              child: Row(
+                children: [
+                  Expanded(
+                    child: ExpansionTile(
+                      title: Row(
+                        children: [
+                            Text(
+                              MyLocalizations.of(context, 'auto_tagging_settings_title'),
+                            ),
```

Comment:
Add `Spacer()` to push the counter to the right

Reply:
done
![image](https://github.com/user-attachments/assets/4a054669-f279-4f4d-ac2f-a438206cd416)